ui-list/src/main/java/com/sfaxdroid/list/pixa/PixaBayViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.sfaxdroid.list.pixa
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sfaxdroid.data.mappers.PixaResponse
+import com.sfaxdroid.domain.GetPixaWallpapersUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+
+@HiltViewModel
+class PixaBayViewModel
```

Comment:
add internal because view model in UI module can be visible only in this module not in app or other module